app/mailers/notification_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class NotificationMailer < ApplicationMailer
+  def release_email
+    @user = params[:user]
+    @grouping = params[:grouping]
+    attachments.inline['markus_logo_big.png'] = File.read(Rails.root.join('app', 'assets', 'images', 'markus_logo_big.png'))
```

Comment:
I feel like it's better to use a remote URL in the email itself (`asset_url('markus_logo_big.png')` or something) rather than attaching the file. Worried about big emails.

Reply:
Hmm. I think a remote url could work. Part of the reason I did inline was because in development my phone wouldn't be able to access my local host. However, the case stands that if markus crashes then the remote url fails. Unless I make the remote URL point to the github hosting of the image. Also, it should be noted the image is 13.8kB (I would think that is small but I guess we're considering bad email clients or poor data connections?)

Reply:
@ignaspa forgot to reply to this in my review.

Ah, you're right that the attachment is small, but I'd still prefer having it remote to keep the email as small as possible. MarkUs might send hundreds of these emails at once. (If MarkUs is down the link won't work either.)

Reply:
Ok